modules/fedimint-wallet-tests/src/bin/wallet-module-tests.rs

```diff
@@ -407,7 +407,7 @@ async fn circular_deposit_test() -> anyhow::Result<()> {
             assert_withdrawal(&send_client, &receive_client, bitcoind, fed).await?;
 
             let fedimint_cli_version = FedimintCli::version_or_default().await;
-            if fedimint_cli_version >= *VERSION_0_8_0_ALPHA {
+            if fedimint_cli_version >= *VERSION_0_7_0_ALPHA {
```

Comment:
That's not technically correct, it just happens to work because we only test with the latest minor release of each major release. Otherwise we'd need to say >0.7.2. Good enough unless we intend to ever run the full suite (unrealistic at this point).

Reply:
Great catch, thanks for flagging! This will throw a wrench in any future releases on `v0.7` since we run against the earlier versions for back-compat. I think it's worth fixing now to save some debugging cycles during the release process https://github.com/fedimint/fedimint/pull/7520